show/interfaces/__init__.py

```diff
@@ -467,8 +476,48 @@ def pm(interfacename, namespace, verbose):
     if namespace is not None:
         cmd += ['-n', str(namespace)]
 
+    cmd += ['-s', window_size, '-i', str(window_index)]
     clicommon.run_command(cmd, display_cmd=verbose)
 
+@pm.command(help="Display current PM data for the specified window size")
+@click.argument('window_size', type=click.Choice(['60sec', '15min', '24hrs']))
+@click.argument('interfacename', required=False)
+@click.option('--namespace', '-n', 'namespace', default=None, show_default=True,
+              type=click.Choice(multi_asic_util.multi_asic_ns_choices()),
+              help='Namespace name or all')
+@click.option('--verbose', is_flag=True, help="Enable verbose output")
+def current(window_size, interfacename, namespace, verbose):
+    """Show interface transceiver pm current subcommand"""
+    ctx = click.get_current_context()
+    cli_pm_helper(window_size, 0, interfacename, namespace, verbose, ctx)
+
+@pm.group(help="Display history PM data for the specified window size")
+@click.argument('window_size', type=click.Choice(['60sec', '15min', '24hrs']))
+def history(window_size):
+    """Show interface transceiver pm history subcommand <window_size>"""
+    ctx = click.get_current_context()
+    ctx.obj = {'window_size': window_size}
+
+@history.command(help="Display history PM data for the specified window index, starting from 1")
```

Comment:
60sec show cumulative pm statistics for the given pm window number from 60sec pm window.​
15min show cumulative pm statistics for the given pm window number from 15min pm window.​
24hrs  show cumulative pm statistics for the given pm window number from 24hrs pm window.​

Reply:
as discussed, updated to Show PM stats of a given window number for the given window size